### PR TITLE
first version

### PR DESCRIPTION
**Pull request by ocobles**
Workshop to extend https://github.com/equinix-labs/pulumi-equinix-fabric-cloud-router-workshop adding connectivity to AWS and covering the multi-cloud use case with a fabric cloud router

- 1. Setup
- 2. Fabric connection to AWS
- 3. Direct Connect configuration
- 4. Configuring Direct Connect to AWS VPC
- 5. Configuring BGP in Equinix Fabric
- 6. IDeploying VM Instances in Both Clouds
- 7. Configuration and Provisioning
- 8. Testing Connectivity between GCP and AWS Instances

### CODE REVIEW

File: docs/parts/conclusion.md [hunk 1]
```diff
@@ -1,22 +1,24 @@
...
+
+* Experiment with additional cloud providers and services to extend your multicloud environment. Integrate platforms like Microsoft Azure or other Equinix Fabric-supported clouds to diversify your infrastructure.
+* Dive deeper into Pulumi's features and capabilities, exploring advanced deployment strategies, infrastructure as code best practices, and leveraging Pulumi's extensive ecosystem of libraries and integrations.
+
 ## Resources
 
-Here are a few other resources to look at to continue your Equinix Metal journey:
+Here are a few other resources to look at to continue your Equinix Fabric journey:
 
 * [Deploy @ Equinix](https://deploy.equinix.com): A one-stop shop for blogs, guides, and plenty of other resources.
```
Comment (ctreatma) [inline docs/parts/conclusion.md:21]: I believe the Deploy site is Metal-specific (although it does have an external Fabric link in the header), but IMO keeping it is a good call (also a good call to remove the other links that are Metal-specific)

File: docs/parts/part2_fabric_connection_to_aws.md [hunk 2]
```diff
@@ -0,0 +1,72 @@
+<!-- See https://squidfunk.github.io/mkdocs-material/reference/ -->
+# Part 2: Fabric connection to AWS
+
+In this section, we'll start by configuring the connection from Equinix Fabric to AWS. fFor now, we'll add the code in the `__main__.py` file, although later on, we'll separate some logic into other files for better readability and maintenance.
```
Comment (ctreatma) [inline docs/parts/part2_fabric_connection_to_aws.md]:
```suggestion
In this section, we'll start by configuring the connection from Equinix Fabric to AWS. For now, we'll add the code in the `__main__.py` file, although later on, we'll separate some logic into other files for better readability and maintenance.
```

File: docs/parts/conclusion.md (hunk 1 above)
Comment (ocobles) [reply docs/parts/conclusion.md:21]: Can we say "Equinix `platform` journey" instead?

Comment (ctreatma) [reply docs/parts/conclusion.md:21]: Yeah that sounds good

**Pull request merged**